data/human/fleets.txt

```diff
@@ -655,6 +699,36 @@ fleet "Large Northern Merchants"
 	variant 1
 		"Scout (Speedy)" 3
 
+fleet "Northern Container Transports"
```

Comment:
Not a fan of such a specific fleet addition like this. Can't such fleet variants just exist in the merchant fleets that spawn where this fleet is being added?